### ISSUE TITLE
Allow "Drag while maximized" feature while custom chrome is enabled

### ISSUE THREAD

**Issue by DingusPingus** [runelite/runelite#14208]
Currently if custom chrome is enabled in the Runelite plugin you cannot drag the Runelite client while it is currently maximized to fit the screen. Also when you are dragging the window around and you try to snap it to the top of the window to maximize it in the current screen it does not work either. These both work when custom chrome is disabled however. I am curious if there is someway for these features to be able to work while the custom chrome is enabled? 

I am on windows 10 if that is of any help.

**Comment by Nightfirecat** [comment]
Snapping would only work with native chrome. What behavior would you expect for allowing dragging while maximized? I assume it would unmaximize to its previous size?

**Comment by DingusPingus** [comment]
yes it would unmaximize to its previous size and be dragging the window with it as well similar to how it works for some other applications like Discord where if you click and hold on the top bar and try to move it when its maximized in the current screen.

**Comment by robmonte** [comment]
@Nightfirecat Yes, typical Windows 10 window snapping behavior would be:
* Drag to the top of the screen to maximize window
* When maximized, drag window to revert back to window-size
* Drag to left/right side to fill in half of the monitor
* Drag to a corner to fill in a quarter of the screen

Discord has Windows snapping working with a custom Chrome, but I don't know if that's because they had a way to maintain the OS feature or if they re-implemented it themselves onto their custom Chrome.
